numpy/f2py/crackfortran.py

```diff
@@ -903,6 +905,16 @@ def _is_intent_callback(vdecl):
     return 0
 
 
+def _resolvetypedefpattern(line):
+    line = markouterparen(line)
+    m1 = typedefpattern.match(line)
```

Comment:
Insert
```python
line = ''.join(line.split())  # removes whitespace
```
before `m1 = ..` to ensure that `bind( c   )`, for instance, is mapped to `bind(c)` .

Reply:
Line added to remove spaces. It indeed simplify the regexp and ensure that `bind(c)` is properly catched and stored.